tests/functional/test_install.py

```diff
@@ -1321,7 +1321,7 @@ def test_cleanup_after_failed_wheel(script, with_wheel):
     # One of the effects of not cleaning up is broken scripts:
     script_py = script.bin_path / "script.py"
     assert script_py.exists(), script_py
-    shebang = open(script_py, 'r').readline().strip()
+    shebang = open(script_py).readline().strip()
```

Comment:
Not related, but probably a good chance to rewrite this to use `with` instead.

Reply:
Better idea: IIRC pathlib has a method to read the contents of a file.

Reply:
nevermind. I see. This is reading only the first line. I'll add a with statement

Reply:
done